save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt

```diff
@@ -35,11 +35,22 @@ import javax.transaction.Transactional
 @RequestMapping("/sandbox/api")
 class SandboxController(
     val configProperties: ConfigProperties,
-    val storage: Storage<SandboxStorageKey>,
+    val storage: SandboxStorage,
     val sandboxExecutionRepository: SandboxExecutionRepository,
     val sandboxUserRepository: SandboxUserRepository,
     val agentsController: AgentsController,
 ) {
+    /**
+     * @param userName
+     * @return list of available files for provided [userName]
+     */
```

Comment:
Shouldn't we use SwaggerDoc here?

Reply:
Probably we can add it for current controller

Reply:
Will you do it?🥺

Reply:
doing it right now